### PR TITLE
[BugFix] Load pagination controls on component initialization

### PR DESCRIPTION
**Pull request by afiune**
### :nut_and_bolt: Description
If you navigate to `/applications` and have 26+ service groups, you don't see the pagination controls. If you change the status filters, then the pagination controls will appear.

**EXTRA:** Found out that we didn't write tests for the new default sorting field `percent_ok`, so I added them here.

### :+1: Definition of Done
Pagination controls should work on the first load of the Applications Tab page.

### :athletic_shoe: Demo Script / Repro Steps

1) Run automate-ui unit tests!
2) Build and start the automate-ui and all automate services.
3) Ingest a few services messages by using the studio helper:
```
[3][default:/src:0]# for i in a b c d e f g h i j k l m n o p q r s t u v w x y z; do
applications_publish_supervisor_message --sup-id $(uuidgen) --name $i
done
```

The UI should show the paginator at the end of the page like this:
image

### :chains: Related Resources
Jira: https://chefio.atlassian.net/browse/A2-883
Closes https://github.com/chef/automate/issues/620

### :white_check_mark: Checklist

- [x] Necessary tests added/updated?
- [x] Necessary docs added/updated?
- [x] Code actually executed?
- [x] Vetting performed (unit tests, lint, etc.)?

### CODE REVIEW

**Comment by afiune** [comment]
Until this point on time, the tests should fail on purpose (test driven development, ah? ), I am working on the fix but wanted to add test to detect the failure!

**Test it all!** 

tenor-119219276

**Comment by afiune** [comment]
Until this point on time, the tests should fail on purpose (test driven development, ah? ), I am working on the fix but wanted to add test to detect the failure!

**Test it all!** 

tenor-119219276

**Comment by afiune** [comment]
Purposely failing tests:
```
HeadlessChrome 0.0.0 (Linux 0.0.0) ServiceGroupsComponent when the page first loads with ServiceGroupsCounts should update the total number of service groups FAILED
--
  | Expected 0 to equal 21.
HeadlessChrome 0.0.0 (Linux 0.0.0) ServiceGroupsComponent when the page first loads with ServiceGroupsCounts should update the total number of service groups FAILED
  | Expected 0 to equal 21.
HeadlessChrome 0.0.0 (Linux 0.0.0) ServiceGroupsComponent when the page first loads with ServiceGroupsCounts and OK status filter update should update the total number of service groups and selected status FAILED
  | Expected undefined to equal 'ok'.
HeadlessChrome 0.0.0 (Linux 0.0.0) ServiceGroupsComponent when the page first loads with ServiceGroupsCounts and OK status filter update should update the total number of service groups and selected status FAILED
  | Expected undefined to equal 'ok'.
```

**Comment by apriofrost** [comment]
Looking good! 
Could we make the distance between the bottom of the last row and the top of the pagination controls to be 24px? Looks like 10px that's same with the gap between the table rows. More space will give a bit more breathing room between the two components.
Also would like to make sure that the pagination controls are included in the tab order after the last table row so that users can use the `tab` key to navigate through the controls.

**Comment by afiune** [comment]
@apriofrost sounds good, since his is a big fix, could we do styling modifications in a different change?

File: components/automate-ui/src/app/pages/service-groups/service-groups.component.spec.ts [hunk 1]
```diff
@@ -69,6 +110,28 @@ describe('ServiceGroupsComponent', () => {
...
+      });
+    });
+
+    it('when set to an allowed value', () => {
+      spyOn(component.router, 'navigate');
+      component.onToggleSort('environment');
+      expect(component.router.navigate).toHaveBeenCalledWith(
+        [], {queryParams: { sortField: ['environment'], sortDirection: ['ASC'] }});
+    });
+
```
Comment (danielsdeleo) [inline components/automate-ui/src/app/pages/service-groups/service-groups.component.spec.ts]: Yay tests! 💯

**Comment by apriofrost** [comment]
@afiune sounds good! thank you!

**Pull request merged**